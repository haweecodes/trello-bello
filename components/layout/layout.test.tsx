import { act, render, screen } from '@testing-library/react';
import WorkSpaceLayout from './index';

describe('workspace layout', () => {
  it('should render navigation section', async () => {
   const {container} = await act( async () => render(<WorkSpaceLayout><p>Show me</p></WorkSpaceLayout>));
    expect(container.firstChild).toMatchSnapshot()
  })
})