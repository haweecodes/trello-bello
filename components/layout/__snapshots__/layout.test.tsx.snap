// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`workspace layout should render navigation section 1`] = `
<section
  class="ant-layout ant-layout-has-sider layoutContainer css-dev-only-do-not-override-acm2ia"
>
  <aside
    class="ant-layout-sider ant-layout-sider-light mainSidebar"
    style="flex: 0 0 200px; max-width: 200px; min-width: 200px; width: 200px;"
  >
    <div
      class="ant-layout-sider-children"
    >
      <ul
        class="ant-menu ant-menu-root ant-menu-vertical ant-menu-light sidebarMenuItem css-dev-only-do-not-override-acm2ia"
        data-menu-list="true"
        role="menu"
        tabindex="0"
      >
        <li
          class="ant-menu-item ant-menu-item-only-child"
          data-menu-id="rc-menu-uuid-test-/workspace-1"
          role="menuitem"
          tabindex="-1"
        >
          <span
            class="ant-menu-title-content"
          >
            <a
              href="/workspace-1"
            >
              Workspace 1
            </a>
          </span>
        </li>
      </ul>
      <div
        aria-hidden="true"
        style="display: none;"
      />
    </div>
  </aside>
  <p>
    Show me
  </p>
</section>
`;
